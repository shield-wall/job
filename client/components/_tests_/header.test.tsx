import renderer from "react-test-renderer";
import Header from "../header";

describe("Header test", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(
      <Header  />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});