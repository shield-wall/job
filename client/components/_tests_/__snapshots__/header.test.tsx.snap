// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should match snapshot 1`] = `
<nav
  className="nav"
>
  <div
    className="nav-container"
  >
    <a
      className="flex items-center"
      href="/"
    >
      <img
        alt="job"
        className="h-6 mr-3 sm:h-9"
        src="https://flowbite.com/docs/images/logo.svg"
      />
      <span
        className="self-center text-xl font-semibold whitespace-nowrap hover:text-blue-700"
      >
        Job
      </span>
    </a>
    <div
      className="w-full md:block md:w-auto"
      id="navbar-default"
    >
      <ul
        className="nav-ul"
      >
        <li>
          <a
            aria-current="page"
            className="nav-main-li"
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="nav-dropDown-li"
            data-dropdown-toggle="dropdownNavbar"
            href="/"
            id="dropdownNavbarLink"
          >
            Blog
            <svg
              aria-hidden="true"
              className="w-5 h-5 ml-1"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
                fill-rule="evenodd"
              />
            </svg>
          </a>
        </li>
        <li>
          <a
            className="nav-li"
            href="/"
          >
            About
          </a>
        </li>
        <li>
          <a
            className="nav-li"
            href="/"
          >
            Post a job
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
